### PR TITLE
SDK 3.0.0 samples and docs

### PR DESCRIPTION
**Pull request by hitesh-kumar-mahour**
## Description
This PR aims at the following:

1. Updated all the samples to use the latest Adobe PDF Services SDK version 3.0.0.
2. Added a sample, ExportPDFToDOCXWithOCROption, to showcase the support of OCR locales for Export PDF.
3. Added new samples, ExportPDFToJPEG and ExportPDFToJPEGZip, to illustrate the retrieval of either a list or a zip archive of JPEG images from the specified PDF file.
4. Added a sample, GetPDFProperties, to illustrate the retrieval of various properties of the specified PDF file.

### CODE REVIEW

File: ExportPDFToDocxWithOCROption/Program.cs [hunk 1]
```diff
@@ -46,17 +47,21 @@ static void Main()
 
                 //Create an ExecutionContext using credentials and create a new operation instance.
                 ExecutionContext executionContext = ExecutionContext.Create(credentials);
-                ExportPDFOperation exportPdfOperation = ExportPDFOperation.CreateNew(ExportPDFTargetFormat.JPEG);
+                ExportPDFOperation exportPdfOperation = ExportPDFOperation.CreateNew(ExportPDFTargetFormat.DOCX);
 
-                // Set operation input from a source file.
-                FileRef sourceFileRef = FileRef.CreateFromLocalFile(@"exportPdfToImageInput.pdf");
+                // Set operation input from a source file
+                FileRef sourceFileRef = FileRef.CreateFromLocalFile(@"exportPdfInput.pdf");
```
Comment (anubhav-mhjn) [inline ExportPDFToDocxWithOCROption/Program.cs:53]: exportPDFInput.pdf

File: ExportPDFToDocxWithOCROption/Program.cs [hunk 2]
```diff
@@ -46,17 +47,21 @@ static void Main()
...
+                // Create a new ExportPDFOptions instance from the specified OCR locale and set it into the operation.
+                ExportPDFOptions exportPdfOptions = new ExportPDFOptions(ExportOCRLocale.EN_US);
+                exportPdfOperation.SetOptions(exportPdfOptions);
+
                 // Execute the operation.
                 FileRef result = exportPdfOperation.Execute(executionContext);
 
                 // Save the result to the specified location.
-                result.SaveAs(Directory.GetCurrentDirectory() + "/output/exportPdfToImageOutput.zip");
+                result.SaveAs(Directory.GetCurrentDirectory() + "/output/exportPdfOutput.docx");
```
Comment (anubhav-mhjn) [inline ExportPDFToDocxWithOCROption/Program.cs:64]: exportPDFWithOCROptionsOutput.docx

File: ExportPDFToJPEGZip/Program.cs [hunk 3]
```diff
@@ -44,17 +49,20 @@ static void Main()
...
+                ExportPDFToImagesOperation exportPDFToImagesOperation = ExportPDFToImagesOperation.CreateNew(ExportPDFToImagesTargetFormat.JPEG);
                 
-                // Provide an input FileRef for the operation
-                FileRef source = FileRef.CreateFromLocalFile(@"pdfPropertiesInput.pdf");
-                pdfPropertiesOperation.SetInput(source);
+                // Set the output type to create zip of images.
+                exportPDFToImagesOperation.SetOutputType(ExportPDFToImagesOutputType.ZIP_OF_IMAGES);
+
+                // Set operation input from a source file.
+                FileRef sourceFileRef = FileRef.CreateFromLocalFile(@"exportPdfToImageInput.pdf");
```
Comment (anubhav-mhjn) [inline ExportPDFToJPEGZip/Program.cs:58]: exportPDFToImageInput.pdf

**Review by anubhav-mhjn — CHANGES_REQUESTED** [review]

File: GetPDFProperties/GetPDFProperties.csproj [hunk 4]
```diff
@@ -3,10 +3,11 @@
     <PropertyGroup>
         <OutputType>Exe</OutputType>
         <TargetFramework>netcoreapp3.1</TargetFramework>
+        <RootNamespace>PDFPropertiesAsFile</RootNamespace>
```
Comment (ankur010592) [inline GetPDFProperties/GetPDFProperties.csproj:6]: Remove this line

**Review by ankur010592 — CHANGES_REQUESTED** [review]

**Review by anubhav-mhjn — APPROVED** [review]

**Review by ankur010592 — APPROVED** [review]

**Pull request merged**